helm/values.yaml

```diff
@@ -311,7 +311,7 @@ opentelemetry-collector:
       enabled: false
     jaeger-thrift:
       enabled: false
-    jaeger-rpc:
+    jaeger-grpc:
```

Comment:
I don't think we even use Jaeger? WildFly's Open Telemetry implementation stopped supporting Jaeger from what I read.

Reply:
thats why I am disabling the jaeger endpoints that are enabled by default... this was just a typo (rpc instead of grpc)